src/content/docs/identityserver/diagnostics/otel.md

```diff
@@ -141,13 +141,16 @@ Note that the pages in the user interface are not IdentityServer endpoints and a
 | endpoint | The type name for the endpoint processor |
 | path     | The path of the request                  |
 
-### Detailed Metrics - Experimental
+### Detailed Metrics
 
 These detailed metrics are instrumented by the IdentityServer middleware and services and track usage of specific
-flows and features. These metrics are created by the meter named "Duende.IdentityServer.Experimental", which is
-the value of the `Duende.IdentityServer.Telemetry.ServiceName.Experimental` constant.
-The definition and tags of these counters may be changed between releases. Once the counters and tags
-are considered stable they will be moved to the `Duende.IdentityServer.Telemetry.ServiceName` meter.
+flows and features.
+
+:::note
+In IdentityServer versions <span data-shb-badge data-shb-badge-variant="default">&lt;7.3</span>, 
```

Comment:
For my own education, can you tell me about the badges? I went looking and also see a Badge component imported elsewhere.

Reply:
The badge component works in the `.mdx` files, but we decided for now to use the literal HTML/CSS in docs pages that are simple Markdown and have only one badge. In theory we could update to .mdx everywhere but they make it easier to shoot yourself in the foot with frontmatter etc. We'll iterate/update as we evolve.

These pages give a good overview of components available in each file type:
https://starlight.astro.build/guides/authoring-content/
https://starlight.astro.build/components/using-components/